packages/app/src/elements/export-dialog.ts

```diff
@@ -81,7 +91,72 @@ export class ExportDialog extends Dialog<void, void> {
 
             case PBES2.value:
                 this.open = false;
-                const password = await prompt($l("Please choose a password to protect this backup with!"), {
+                if (!password) {
```

Comment:
This is the same password logic, but inside this `if`, and looks funky in the git diff, as if it was completely added.

Reply:
Why not just require the `password` argument and remove this whole code block? There's really no reason to have this exact same code in there twice.

Reply:
You're right, I can simply force this earlier.